Nos/Views/NoteComposer/NoteComposer.swift

```diff
@@ -201,15 +227,40 @@ struct NoteComposer: View {
         }
         isPresented = false
     }
+    
+    private func loadQuotedNote() {
+        guard let quotedNoteID else {
+            return
+        }
+        
+        @Dependency(\.persistenceController) var persistenceController
+        quotedNote = try? Event.findOrCreateStubBy(
+            id: quotedNoteID,
+            context: persistenceController.viewContext
+        )
+    }
 
+    private var postText: AttributedString {
+        var text = editingController.text ?? ""
+        if let noteLink = quotedNote?.bech32NoteID {
+            if !text.characters.isEmpty {
+                text += AttributedString("\n\n")
+            }
+            text += AttributedString("nostr:\(noteLink)")
+        }
+        return text
+    }
+    
     // swiftlint:disable:next function_body_length
     private func publishPost() async {
         guard let keyPair = currentUser.keyPair, let author = currentUser.author else {
             Log.error("Cannot post without a keypair")
             return
         }
         
-        guard let text = editingController.text else {
+        let text = postText
+        
+        guard !text.characters.isEmpty else {
```

Comment:
This looks like a slight change to the behavior here. I'm not sure if that's good, bad, or neutral, but I think @mplorentz would be able to say for sure.

Reply:
The original check didn't seem very helpful to me because it only verifies that a textView has been associated with the `NoteEditorController`, not that the post will be empty.

The reason I had to do this was because when the user quotes another note, I don't immediately put the link into the content they're editing. It gets appended once they post because I need to know whether to add a couple newlines before the note link or not. And if you don't add your own comment and you want to post *only* the note link, then the content will be empty.

I just saw that there is an `isEmpty` getter on the `NoteEditorController`. So I can use that. It will be clearer at this call site what it means, and it's also already used to mark the Post button disabled. In fact, I'll make a `canPost` getter that both the Post button and this guard can use.